examples/models/toolbars2.py

```diff
@@ -1,3 +1,11 @@
+''' This example shows multiple ways to place the toolbar with respect to a grid of up to four plots.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.layouts.column, bokeh.layouts.gridplot, bokeh.layouts.row, bokeh.plotting.figure, bokeh.plotting.show
+    :refs: :ref:`ug_interaction_tools_toolbar`
+    :keywords: tools, toolbar, mkplot, mkgrid, layout
```

Comment:
`mkgrid` and `mkplot` are helpers that exist only in this example, they are not relevant to anything else in a wider context
```suggestion
    :keywords: tools, toolbar, layout
```